bundles/runtime/org.eclipse.fx.ui.controls/src/org/eclipse/fx/ui/controls/tabpane/DndTabPane.java

```diff
@@ -0,0 +1,153 @@
+package org.eclipse.fx.ui.controls.tabpane;
+
+import static org.eclipse.fx.ui.controls.tabpane.DndTabPaneFactory.setup;
+import static org.eclipse.fx.ui.controls.tabpane.DndTabPaneFactory.teardown;
+
+import java.util.function.Consumer;
+
+import org.eclipse.fx.ui.controls.tabpane.DndTabPaneFactory.DragSetup;
+import org.eclipse.fx.ui.controls.tabpane.DndTabPaneFactory.FeedbackType;
+import org.eclipse.fx.ui.controls.tabpane.skin.DnDTabPaneSkinHookerFullDrag;
+import org.eclipse.fx.ui.controls.tabpane.skin.DndTabPaneSkinHooker;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.scene.control.Skin;
+import javafx.scene.control.SkinBase;
+import javafx.scene.control.TabPane;
+
+/**
+ * A TabPane with Drag and Drop Support
+ *
+ * @author Adrodoc55
```

Comment:
@tomsontom are there no releases planned before 3.0.0? Is there a rough plan on when I can expect my change to get released (because I'd like to use it :))?